python/run-tests.py

```diff
@@ -22,22 +22,22 @@
 from os import path
 
 
-def test(root_dir, package):
-    # Run all of the test under test/python directory, each of them
+def test(root_dir, test_path, packages):
```

Comment:
nit: I would name both for consistency either `root_dir/test_dir` or `root_path/test_path`. Potentially the later.